airbyte-cdk/bulk/toolkits/load-low-code/src/main/kotlin/io/airbyte/cdk/load/checker/HttpRequestChecker.kt

```diff
@@ -0,0 +1,14 @@
+package io.airbyte.cdk.load.checker
+
+import io.airbyte.cdk.load.check.DestinationChecker
+import io.airbyte.cdk.load.command.DestinationConfiguration
+import io.airbyte.cdk.load.http.HttpRequester
+
+class HttpRequestChecker<C : DestinationConfiguration>(
+    private val requester: HttpRequester
+): DestinationChecker<C> {
+    override fun check(config: C) {
+        val response = requester.send()
+        response.use { assert(it.statusCode == 200, {"Expected status code to be 200 but was ${it.statusCode}"}) }
```

Comment:
At some point, it could be nice to validate the authorizations associated with the credentials as it would be better UX than to fail on check than on sync but it's always a complex thing during the check because we don't know which streams the user needs to access...